src/GuestList/GuestListMainPage.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Container, Table } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { renderRsvpStats, createListOfGuests } from './utils';
+
+const GuestListMainPage = () => (
+  <>
+    <Container fluid="md" className="header">
+      <Container>
+        <h1>Betrothed</h1>
+      </Container>
```

Comment:
"While containers can be nested, most layouts do not require a nested container." While I think it's fine that they are nested, boostrap docs think it might not be best practice. 

As for betrothed header and navbar, react router has a way of including things that are used globally by keeping them outside of the router, not sure how it'll work because some pages have the logo/navbar/footer but something to look into